q2_taxa/plugin_setup.py

```diff
@@ -48,6 +61,10 @@
         'table': FeatureTable[Frequency]
     },
     parameters={'metadata': qiime2.plugin.Metadata},
+    input_descriptions={
+        'taxonomy': 'Taxonomic annotations for all features.',
+        'table': 'Feature table containing counts of features by sample.'},
```

Comment:
I think *containing counts of features by sample* isn't necessary here b/c that's described by the semantic type (and the type name uses the standard "frequency" terminology instead of "counts"). What about: *Feature table to visualize at various taxonomic levels.* ?